lib/mondo/account.rb

```diff
@@ -1,8 +1,12 @@
 module Mondo
   class Account < Resource
 
-    attr_accessor :id, :description, :raw_data
+    attr_accessor :id, :description, :raw_data, :sort_code, :account_number
 
     date_accessor :created
+
+    def balance
+      self.client.balance
```

Comment:
I feel like we should explicitly pass an account id here. If you are using a set client without an account_id, and you loop over multiple accounts, it will always just return the balance for the first one.